### ISSUE TITLE
Cannot subtract tuple from integer, also error message is wrong

### ISSUE THREAD

**Issue by meijeru** [red/red#2703]
```
>> about
Red for Windows version 0.6.2 built 20-May-2017/19:45:33+2:00
>> 1 + 2.3.4
== 3.4.5
>> 1 - 2.3.4
*** Script Error: - does not allow percent! for its value2 argument
*** Where: -
```
It is not that negative tuple elements are not allowed - when subtracting an integer from a tuple, the result is kept between 0 and 255 for each element.

**Comment by greggirwin** [comment]
Same comment as for #2702.

**Comment by dockimbel** [comment]
You cannot subtract a tuple from an integer, as the tuple needs to be negated, and that is not a supported operation from tuples (the components have to remain positive, unlike in pairs).

**Comment by meijeru** [comment]
I accept the reasoning. Still, if the error message has not been corrected, this should not yet have been dismissed!

**Comment by dockimbel** [comment]
Indeed, there was a correction for the error message, but it was done in the fix for #2702.
